Functions/Coverage.ps1

```diff
@@ -69,14 +69,21 @@ function Get-CoverageInfoFromUserInput
 
 function New-CoverageInfo
 {
-    param ([string] $Path, [string] $Class = $null, [string] $Function = $null, [int] $StartLine = 0, [int] $EndLine = 0)
+    param ([string] $Path, [string] $Class = $null, [string] $Function = $null, [int] $StartLine = 0, [int] $EndLine = 0, [bool] $IncludeTests = $false)
+
+    # If user did not specify IncludeTests-value then auto-detect based on path
+    if ($PSBoundParameters.ContainsKey("IncludeTests") -eq $false) {
```

Comment:
This is a bit confusing to me. I can see that `Get-CoverageInfoFromUserInput` calls `New-CoverageInfo` in two places. Once directly and once through the dictionary parser. In the dictionary parser the bound parameter will always be specified, so this only applies to the direct call. 

The direct call will never specify `IncludeTests` because in the else branch we got just a string. So imho this should move into the `Get-CoverageInfoFromUserInput`, and the IncludeTestsParemeter should become mandatory.

Reply:
Result is the same (autodetect only used for path input), so I agree that it should be moved to keep it clean.